src/main/java/com/internet/shop/controllers/ProductAddController.java

```diff
@@ -0,0 +1,38 @@
+package com.internet.shop.controllers;
+
+import com.internet.shop.lib.Injector;
+import com.internet.shop.models.Product;
+import com.internet.shop.services.ProductService;
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class ProductAddController extends HttpServlet {
+    private static final Injector injector =
+            Injector.getInstance("com.internet.shop");
+    private ProductService productService =
+            (ProductService) injector.getInstance(ProductService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+
+        req.getRequestDispatcher("/WEB-INF/views/product/add.jsp").forward(req, resp);
+    }
+
+    @Override
+    protected void doPost(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        String name = req.getParameter("name");
+        String price = req.getParameter("price");
+        if (name.length() >= 1 && price.length() >= 1) {
```

Comment:
better to do it at the jsp using HTML tags - https://www.w3schools.com/html/html_form_input_types.asp (Input Restrictions, Input Type Range)